app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsFragment.java

```diff
@@ -117,7 +117,6 @@ private void initEmptyView() {
     }
 
     private Subscription fetchAndSave() {
-        showLoadingView();
```

Comment:
It seems ok.. (using intentional 3G network)


![bullheadmmb29qa1347002162016024417](https://cloud.githubusercontent.com/assets/920911/13056224/622bf8e8-d457-11e5-8247-2987dc28ce69.gif)